packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/secretValuesWalkthrough.ts

```diff
@@ -176,12 +176,20 @@ const enterSecretName = async (invalidNames: string[]) =>
 
 const secretValueDefaultMessage = (secretName: string) => `Enter the value for ${secretName}:`;
 
+export const secretValueValidator = () => (input?: string) => {
```

Comment:
I think the `validate` parameter only needs the function, and doesn't need the top level one.
```suggestion
export const secretValueValidator = (input?: string) => {
```

Reply:
Sure, that seems better